vars/parseStageInfo.groovy

```diff
@@ -215,6 +215,10 @@ def call(Map config = [:]) {
         result['node_count'] = 5
         cluster_size = 'hw,medium'
         result['pragma_suffix'] = '-hw-medium'
+      } else if (stage_name.contains('Xlarge')) {
+        result['node_count'] = 24
+        cluster_size = 'hw,xlarge'
+        result['pragma_suffix'] = '-hw-xlarge'
```

Comment:
I'm kind of wishing we didn't use _pretty_ names here like _Small_, _Medium_ and _Large_.  And now _XLarge_.  What happens if there is a 15-node cluster created?  Is it called _BiggerThanLargeButNotXLarge_?

Maybe they should have just been called Hardware3, Hardware5, Hardware9 and now Hardware24.  Or something like that.